xbmc/video/windows/GUIWindowFullScreenDefines.h

```diff
@@ -0,0 +1,36 @@
+/*
+ *      Copyright (C) 2017 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this Program; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#define BLUE_BAR                          0
```

Comment:
I know this PR didn't introduce this define, but what is "blue bar" supposed to be? The header? Might be good to rename this define into something more descriptive and especially not related to any skin/design choice.

Reply:
How many skinners read source code? I completely agree that this should be documented, but I'm not sure the source is the right place for skinning stuff. From a programmer's side it doesn't really matter, just another anonymous variable in a sea of many.